frontend/src/Components/ClassStatus.tsx

```diff
@@ -127,13 +127,11 @@ export default function ClassStatus({
             return;
         setDropdownOpen(false);
         setSelectedModifyOption(option);
+        setTimeout(() => showModal(modifyClassRef), 0);
+        showModal(modifyClassRef);
```

Comment:
@corypride This looks like a code smell, calling `showModal` two lines apart, one with a setTimeout of 0ms. Is this the final solution? If it is, then can you add a comment why this is necessary?